microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -569,8 +585,8 @@
                     }
 
                     state.dataset_type.value = '16S';
-                    state.dataset_site.value = 'gut';
-                    state.dataset_input.value = 'tmi-16S-gut';
+                    state.dataset_site.value = body_site;
```

Comment:
I like that state.dataset_site.value is now being set with a variable instead of hard-coded! Since the value being stored in state.dataset_input.value can be expressed as 'tmi-' + state.dateset_type.value + '-' + body_site , it would be great to go a step farther and move the line to set state.dataset_site.value = body_site and the alternate line above to set state.dataset_input.value OUT of the if/else, because they will be the same for both branches.